client/src/test/scala/com/crobox/clickhouse/ClickhouseClientAsyncSpec.scala

```diff
@@ -58,4 +59,12 @@ abstract class ClickhouseClientAsyncSpec(val config: Config = ConfigFactory.load
         succeed
       })
   }
+
+  def compressGzip(content: String): Array[Byte] = {
+    val arrOutputStream = new ByteArrayOutputStream()
+    val zipOutputStream = new GZIPOutputStream(arrOutputStream)
+    zipOutputStream.write(content.getBytes)
```

Comment:
Can we make sure this is using the `UTF-8` encoding? I think by default it will use the system one which might be causing encoding issues

Reply:
Corrected, thanks